src/axom/inlet/ConduitReader.cpp

```diff
@@ -471,9 +471,13 @@ ReaderResult ConduitReader::getArray(const std::string& id,
     {
       detail::arrayToMap(node.as_double_array(), values);
     }
-    else if(node.dtype().is_integer())
+    else if(node.dtype().is_int32())
     {
-      detail::arrayToMap(node.as_long_array(), values);
+      detail::arrayToMap(node.as_int32_array(), values);
+    }
+    else if(node.dtype().is_int64())
+    {
+      detail::arrayToMap(node.as_int64_array(), values);
```

Comment:
@joshessman-llnl  and @cyrush  -- please confirm this change.  It resolved two failing tests in msvc when building with the `x64-windows` "triplet"

Reply:
new logic looks more robust to me. 

`is_integer`  matches to any integer type (both signed/unsigned and across all bitwidths). `as_zzz` requires you know the exact type.